bondpad.py

```diff
@@ -34,6 +40,7 @@ class Bondpad(Component):
     _defaults['taper_length'] = 300
     _defaults['launcher_padding'] = 167.44
     _defaults['bond_pad_length'] = 350
+    _defaults['spec'] = 'auto'
```

Comment:
I'd suggest changing the name to "type" or something. "spec" means a very specific thing in the lab